.gitignore

```diff
@@ -83,3 +99,4 @@ lint/generated/
 lint/outputs/
 lint/tmp/
 # lint/reports/
+>>>>>>> 47c640aaeb2cf30a8a4ca9dcdafdf8d39c84a450
```

Comment:
YOu have to remove this `conflict lines`

Reply:
also, you need to ignore some files and directories (/.ideas/*). (it's already declared in the top of .gitignore file)